tutor/src/flux/student-id.coffee

```diff
@@ -0,0 +1,39 @@
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+{AppActions} = require './app'
+_ = require 'lodash'
+
+ERROR_MAP = {
+  student_identifier_has_already_been_taken:
+    'The provided student ID has already been used in this course. ' +
+    'Please try again or contact your instructor.'
+  blank_student_identifer:
+    'The student ID field cannot be left blank. Please enter your student ID.'
+}
+
+
+StudentIdConfig = {
+
+  _failed: (error, id) ->
+    @_errors[id] = error?.data?.errors
+    AppActions.resetServerErrors()
```

Comment:
are you doing this to prevent the default error handler?  we've been doing that like [this](https://github.com/openstax/tutor-js/blob/master/tutor/src/api.coffee#L193), where `displayError` prevents the server error modal, and `handleError` allows for endpoint specific error handling